mlir/utils/performance/conv-configs

```diff
@@ -0,0 +1,17 @@
+conv -F 1 -f NCHWG -I NCHWG -O NCHWG -n 2 -c 1280 -H 32 -W 32 -k 640 -y 1 -x 1 -p 0 -q 0 -u 1 -v 1 -l 1 -j 1 -g 1
```

Comment:
IIUC, this file is stands as a generic conv-configs. Therefore, it would be best that we put a comment indicating the source of these configs (like in gemm configs).

Something as future work for us :
Also noting we have 4 other similiar files which are defined per model (resnet50 and resnext101). Though I dont personally see that it is necessary bring them under here in the PR, it would be good to set that as the direction we want to go (prolly in a future PR). The 4 files seems to be coming from : model x {nightly,pr} combinations. If we are agreeing on unifying the filenames be to model agnostic, we should probably be renaming these to be two files along the lines (or similiar) : 

* conv_configs
* conv_configs_pr_subset

Reply:
(I dont see resnext101 configs even used, if I m not missing anything)